rtmbot/core.py

```diff
@@ -61,6 +67,12 @@ def connect(self):
     def _start(self):
         self.connect()
         self.load_plugins()
+        for plugin in self.bot_plugins:
+            try:
+                self._dbg("Registering jobs for {}".format(plugin.name))
+                plugin.register_jobs()
+            except:
```

Comment:
Cool call on not just smothering this error. I have to catch a NotImplementedError here first because that is expected behavior if this plugin has no jobs written into it, but I can catch those and pass otherwise, I'll output the error.